qoffer-infrastructure/src/main/groovy/life/qbic/Controller.groovy

```diff
@@ -0,0 +1,14 @@
+package life.qbic
+
+/**
+ * Controller class responsible for the data flow into qoffer-2
+ *
+ * This class creates instances of qoffer-2 classes and injects them as described in the architectural draft.
+ *
+ *
+ * @since: 1.0
+ * @author: Jennifer Bödker
+ * @see <a href="https://github.com/qbicsoftware/qoffer-2-portlet/tree/master/docs/"</a>
```

Comment:
```suggestion
 * @see <a href="https://github.com/qbicsoftware/qoffer-2-portlet/tree/master/docs/"</a>
```
Why do you link to the master docs here? I think it would make more sense to stay inside the javadoc.

Reply:
The idea was to link the architectural model of qoffer-2.0 here as soon as its uploaded to master/docs. We also include 
external documents in javadocs at the portlet generation via qube(see comment qOfferPortlet)

Reply:
The idea is good but the architecture diagram does not belong inside the JavaDoc. I rather expect it to be linked to the repository readme.

Reply:
Alright, then i'll remove it. Thanks! :)